tests/oss-fuzz/pdu_parse_target.c

```diff
@@ -4,6 +4,7 @@ int
 LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     coap_pdu_t *pdu = coap_pdu_init(0, 0, 0, size);
     if (pdu) {
+        coap_set_show_pdu_output(0);
```

Comment:
Why do you want the PDU output with log prefix? It is not that the timestamp or log level provide useful information here...

Reply:
I misunderstood this API. Have removed this line now.